homeassistant/components/sensor/zwave.py

```diff
@@ -56,8 +60,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         # Check workaround mappings for specific devices.
         if specific_sensor_key in DEVICE_MAPPINGS:
             if DEVICE_MAPPINGS[specific_sensor_key] == WORKAROUND_IGNORE:
+                logger.info('Ignoring device: %s, Index: %s',
+                            zwave._value_name(value), value.index)
                 return
 
+    if value.label == 'Previous Reading':
+        # Power measurement devices may export a 'Previous Reading' value
+        # This can be problematic, as can be seen here:
+        # https://github.com/home-assistant/home-assistant/issues/3509
+        logger.info('Ignoring device: %s', zwave._value_name(value))
```

Comment:
@lukas-hetzenecker I just wondered if this may affect the reporting groups options. Specifically the Aeotec Energy Meter has these [options](https://aeotec.freshdesk.com/support/solutions/articles/6000115095-setting-parameter-101-103-for-the-hem-g2-dsb28-). If I set a specific value for group reporting... for example, set group 1 to value 15 which gives total readings to report group 1. When a value is set that combines multiple readings to a report group I see the previous readings sensor rapidly change between displaying each of those readings for the report group values requested. It probably is better to ignore these because there are better options. But I wasn't sure if we could just make it display the other values as attributes of the sensor instead.